groups/migrations/0001_initial.py

```diff
@@ -0,0 +1,91 @@
+# Generated by Django 4.2.6 on 2023-10-20 05:20
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Group",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("created", models.DateTimeField(auto_now_add=True)),
+                ("name", models.CharField(max_length=100)),
+                ("active", models.BooleanField(default=True)),
+                (
+                    "schedule",
+                    models.CharField(
+                        choices=[
+                            ("BIWEEKLY", "Biweekly"),
+                            ("QUARTERLY", "Quarterly"),
+                            ("MONTHLY", "Monthly"),
+                        ],
+                        default="BIWEEKLY",
+                    ),
+                ),
+                ("logo_url", models.TextField()),
```

Comment:
groups can have logos? Is this going to hold like a link to the image?

Reply:
im dumb and forgot this was a thing. good catch and adding that in!

Reply:
Yep its sort of like a banner you can have on top of your newsletter. I have it as not required for now, but it'll probably be a link to an image we create when we process someones image upload.